[Fix] Issue 866. DNC Body AF quest progression.

hooksta4 opened the pull request:
Fix for issue #866

<!-- remove space and place 'x' mark between square [] brackets or click the checkbox after saving to affirm: -->
**_I affirm:_**
- [x] I have paid attention to this example and will edit again if need be to not break the formatting, or I will be ignored
- [x] that I agree to LandSandBoat's [Limited Contributor License Agreement](https://github.com/LandSandBoat/server/blob/base/.github/CONTRIBUTOR_AGREEMENT.md), as written on this date
- [x] that I have **read the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md)**
- [x] that I've _**tested my code and things my code changed**_ since the last commit in the PR, and will test after any later commits

hookstar committed:
Update Harmodios.lua

Fix for issue #866

Xaver-DaRed commented on scripts/zones/Bastok_Markets/npcs/Harmodios.lua:
```diff
@@ -47,9 +47,9 @@ entity.onEventFinish = function(player, csid, option)
     elseif csid == 490 then
+        player:setCharVar("comebackQueenCS", 2)
         player:startEvent(491)
     elseif csid == 491 then
```
This change leaves this block empty.
Should be deleted

Xaver-DaRed commented on scripts/zones/Bastok_Markets/npcs/Harmodios.lua:
```diff
@@ -47,9 +47,9 @@ entity.onEventFinish = function(player, csid, option)
         player:startEvent(491)
     elseif csid == 491 then
-        player:setCharVar("comebackQueenCS", 2)
     end
```
This change leaves this block empty.
```lua
elseif csid == 491 then
end
```
Should be deleted

hooksta4 replied:
Guess could take out the entire thing then yea?

zach2good commented:
Can we get a more descriptive commit message and PR title, please?

hookstar committed:
DNC AF Body Fix Issue#866

Clean up of where CS is called; removed elseif statement for secondary CS that doesnt have the need to be there.

Xaver-DaRed commented on scripts/zones/Bastok_Markets/npcs/Harmodios.lua:
```diff
@@ -48,8 +48,7 @@ entity.onEventFinish = function(player, csid, option)
         player:startEvent(491)
-    elseif csid == 491 then
-        player:setCharVar("comebackQueenCS", 2)
+		player:setCharVar("comebackQueenCS", 2)
```
Align this line properly and this is good to go.
No tabs. 4 spaces.

hooksta4 replied:
okie thanks

hooksta4 replied:
deleted tab that mysteriously got in there and replaced with spaces to line up.

Xaver-DaRed commented:
Took the freedom to change the PR title, to be more descriptive.
On another note, for a future next time, those boxes in the PR description have an extra space in it, breaking the checkmark functionality

hookstar committed:
Update Harmodios.lua

hooksta4 commented:
> Took the freedom to change the PR title, to be more descriptive. On another note, for a future next time, those boxes in the PR description have an extra space in it, breaking the checkmark functionality

Still not super good at this.. the boxes?

Xaver-DaRed commented:
I mean this.
![imagen](https://user-images.githubusercontent.com/60053999/141348410-d16e5ffd-b9f8-4d59-a392-1a214ced4200.png)

hooksta4 commented:
> I mean this. ![imagen](https://user-images.githubusercontent.com/60053999/141348410-d16e5ffd-b9f8-4d59-a392-1a214ced4200.png)

fair enough.. will be more mindful moving forward. Thanks

Xaver-DaRed reviewed (approved):
Tested. Confirmed working correctly.
